src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -273,24 +277,33 @@ private LogEventLevel ValidateLogLevel(string levelText)
             }
         }
 
-        private int ValidateConcurrentTestrunners(int maxConcurrentTestRunners)
+        private int ValidateConcurrentTestrunners(int? maxConcurrentTestRunners)
         {
+            var safeProcessorCount = Math.Max(Environment.ProcessorCount / 2, 1);
+
+            if (maxConcurrentTestRunners.HasValue)
```

Comment:
Is this right? If a value has been passed by the user we still use the safe value. I think we should use the safe value if the user did NOT pass a value

Reply:
Woops, my bad